sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -13,14 +13,20 @@
   `_OTELRESOURCE_` metrics.
   ([#41761](https://github.com/Azure/azure-sdk-for-net/pull/41761))
 
+* Fixed an issue where tags associated with Exceptions, specifically those logged using
```

Comment:
> specifically those logged using
>   [Activity.RecordException()

This is not correct. This exporter has no idea if the exception is added via RecordException or AddActivityEvent.